api.md

```diff
@@ -3125,16 +3098,6 @@ total_successes        | 1
 total_failures         | 0
 ```
 ---
-## timescaledb.license_key [](timescaledb_license-key)
```

Comment:
This will remain, but the parameter will probably be called `timescaledb.license`. Is the plan to add back later?

Reply:
To me it sounds that it will be new API. `timescaledb.license` is not `timescaledb.license_key` and `timescaledb.license` will not be a view. Is there already an example in regression tests how `timescaledb.license` is used?

Reply:
It is not a new API. The proposal is to rename the GUC from `timescaledb.license_key` to `timescaledb.license`. The view will be removed since it no longer provides any additional information over the GUC.

Reply:
I see. I was confused. Is there example of output?

Reply:
The section of using `timescaledb.licesne` needs to be written. I suggest to do it separately when the implementation is done.
@erimatnor Do you agree? Is your comment resolved?